test/test_plugin_systemd.rb

```diff
@@ -84,15 +86,18 @@ def test_systemd_cluster_notify
   def assert_restarts_with_systemd(signal, workers: 2)
     skip_unless(:fork) unless workers.zero?
     cli_server "-w#{workers} test/rackup/hello.ru", env: @env
+    get_worker_pids(0, workers) if workers == 2
```

Comment:
> So we do this get_worker_pids call to wait / ensure the server has booted far enough

It's helpful, because they can take some time on GHA.  Or, they're better than 'Ctrl-C'.  Depends on the test.

Also, with some of the restart tests, we want to compare the pids of processes the requests run in.  Or, the data in the return is used...

JFYI, much of the day I'm AFK.  Or checking 'messages' very intermittently...

Thanks, Greg